docs-chef-io/content/inspec/resources/parse_config_file.md

```diff
@@ -50,6 +50,13 @@ where each test
 - May run a command to `stdout`, and then run the test against that output
 - May use options to define how configuration data is to be parsed
 
+### Option names containing dots
+
+A possibly unexpected behaviour may occur when attempting to access option names containing dots with `its()`. A known behaviour, there are two ways to work around it:
+
+* Access the option by using the `params` attribute of the returned resource object
+* Since 4.23.9, `its` can be used by passing the option name in a single-element array (cf. examples)
```

Comment:
```suggestion
* Since 4.23.9, `its` can be used by passing the option name in a single-element array. See `parse_config_file`examples.
```

Reply:
After this PR merges, I will insert an internal reference link to the `parse config file` Examples section as a part of a link clean-up project in the docs. So know that this is planned and won't be omitted in the long term.

Reply:
I bumped the version number but it should be fixed once the PR is actually merged.